src/app/core/entity-components/entity-details/form/form.component.html

```diff
@@ -80,29 +80,36 @@
                 </mat-checkbox>
 
                 <div *ngIf="row.input === 'age'">
-                    <mat-form-field style="width: 50px;">
-                        <input
-                                matInput
-                                placeholder="Age"
-                                type="number"
-                                [value]="entity?.age"
-                                [disabled]="true"
-                        />
-                    </mat-form-field>
+                    
                     <mat-form-field style="width: 120px;">
                         <input
                                 matInput
                                 [formControlName]="row.id"
                                 [placeholder]="row.placeholder"
                                 [matDatepicker]="dateOfBirthDatepicker"
                                 [disabled]="!editing"
+                                #dateOfBirth
                         />
                         <mat-datepicker-toggle
                                 matSuffix
                                 [for]="dateOfBirthDatepicker"
                         ></mat-datepicker-toggle>
                         <mat-datepicker #dateOfBirthDatepicker></mat-datepicker>
                     </mat-form-field>
+                    <mat-form-field style="width: 50px;">
+                        <div
+                            matTooltip="This field is read-only. Edit Date of Birth to change age. Select Jan 1st if you only know the year of birth."
+                            matTooltipPosition="above"
```

Comment:
Add `[matTooltipDisabled]=!editing` to disable the tooltip when the user is not in editing mode.